lix/packages/client/src/git/commit.ts

```diff
@@ -18,7 +20,68 @@ type PartialEntry = {
 	oid: string | undefined
 }
 
-export async function commit({
+export const isoCommit = (
+	ctx: RepoContext,
+	{ author, message }: { author: any; message: string }
+) =>
+	originalIsoGitCommit({
+		fs: ctx.rawFs,
+		dir: ctx.dir,
+		cache: ctx.cache,
+		author: author || ctx.author,
+		message: message,
+	})
+
+export async function commit(
+	ctx: RepoContext,
+	state: RepoState,
+	{
+		author: overrideAuthor,
+		message,
+		include,
+	}: // TODO: exclude,
+	{
+		author?: any
+		message: string
+		include: string[]
+		// exclude: string[]
+	}
+) {
+	if (include) {
```

Comment:
include is non optional at the moment - gues this if is not needed?

Reply:
this will be optional when adding default behaviour